modules/javafx.web/src/main/native/Source/ThirdParty/libxslt/src/CMakeLists.txt

```diff
@@ -0,0 +1,598 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
I don't think we use this file in our build. We should remove it if not.

Reply:
this [CMakeLists.txt] not required , i have removed